src/Servers/IIS/IIS/src/IISServerOptions.cs

```diff
@@ -65,5 +65,10 @@ public class IISServerOptions
                 _maxRequestBodySize = value;
             }
         }
+
+        /// <summary>
+        /// Treat request headers as Latin-1 or ISO/IEC 8859-1 instead of UTF-8.
+        /// </summary>
+        public bool Latin1RequestHeaders { get; set; } = false;
```

Comment:
```suggestion
        public bool Latin1RequestHeaders { get; set; }
```
`false` is the default, or do you want to put explicit emphasis on this?